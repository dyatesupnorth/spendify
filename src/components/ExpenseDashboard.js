import React from "react";
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';

const ExpenseDashboard = () => (
    <div>
    <p>This is from my dashboard</p>
    <ExpenseList />
    <ExpenseListFilters />
    </div>
)
export default ExpenseDashboard;