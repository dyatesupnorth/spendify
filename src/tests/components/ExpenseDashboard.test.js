import React from 'react';
import {shallow} from 'enzyme';
import ExpenseDashboard from '../../components/ExpenseDashboard';

test('should render Header correctly', () => {
    const wrapper = shallow(<ExpenseDashboard />);
    expect(wrapper).toMatchSnapshot();
    //expect(wrapper.find('h1').length).toBe(1)
//   const renderer = new ReactShallowRenderer();
//   renderer.render(<Header />);
//   expect(renderer.getRenderOutput()).toMatchSnapshot();
});
