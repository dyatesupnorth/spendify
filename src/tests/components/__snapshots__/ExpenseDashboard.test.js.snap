// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div>
  <p>
    This is from my dashboard
  </p>
  <Connect(ExpenseList) />
  <Connect(ExpenseListFilters) />
</div>
`;
